src/Plugin/FieldEncryptMap/CoreStrings.php

```diff
@@ -0,0 +1,60 @@
+<?php
+/**
+ * @file
+ * Contains \Drupal\field_encrypt\Plugin\FieldEncryptMap\CoreStrings.
+ */
+
+namespace Drupal\field_encrypt\Plugin\FieldEncryptMap;
+
+use Drupal\field_encrypt\FieldEncryptMapBase;
+
+/**
+ * Provides a Field Encrypt Map.
+ *
+ * @FieldEncryptMap(
+ *   id = "corestrings",
+ *   label = @Translation("Core Strings")
+ * )
+ */
+class CoreStrings extends FieldEncryptMapBase {
+
+  /**
+   * @var $stringService
+   */
+  public $stringService;
+
+  public function __construct($configuration, $plugin_id, $plugin_definitions) {
+    parent::__construct($configuration, $plugin_id, $plugin_definitions);
+
+    // The Encrypt modules string encryption service.
+    $this->stringService = $this->container->get('encryption');
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getMap() {
+    return [
+      'string' => [
+        'value' => $this->stringService,
```

Comment:
@nerdstein, the encryption I saw for non-strings will involve updating the field storage which is a lot more work. Not sure if it's necessary, so I don't intend to do that work. Any reason why this approach wouldn't work? Any specifics should be handled by the service.